### PR TITLE
highlight the importance of placing one's comments at the beginning of

### PR DESCRIPTION
**Pull request by jsteemann**
each line

### CODE REVIEW

File: 3.4/administration-configuration.md [hunk 1]
```diff
@@ -122,7 +122,9 @@ log.level=startup=trace
 {% endhint %}
 
 Comments can be placed in the configuration file by placing one or more
-hash symbols `#` at the beginning of a line.
+hash symbols `#` at the beginning of a line. Comments that are placed in
+other places (i.e. not at the beginning of a line are unsupported and should 
+be avoided to ensure correct parsing of the startup options as intended).
```
Comment (goedderz) [inline 3.4/administration-configuration.md]: That should probably read 

> ... places (i.e. not at the beginning of a line are unsupported) and ... as intended

, i.e. the closing parentheses slipped?

File: 3.5/administration-configuration.md [hunk 2]
```diff
@@ -122,7 +122,9 @@ log.level=startup=trace
 {% endhint %}
 
 Comments can be placed in the configuration file by placing one or more
-hash symbols `#` at the beginning of a line.
+hash symbols `#` at the beginning of a line. Comments that are placed in
+other places (i.e. not at the beginning of a line are unsupported and should 
+be avoided to ensure correct parsing of the startup options as intended).
```
Comment (goedderz) [inline 3.5/administration-configuration.md]: As above.

File: 3.4/administration-configuration.md (hunk 1 above)
Comment (jsteemann) [reply 3.4/administration-configuration.md]: true, fixed.

File: 3.5/administration-configuration.md (hunk 2 above)
Comment (jsteemann) [reply 3.5/administration-configuration.md]: fixed as well

**Pull request merged**